README.md

```diff
@@ -15,7 +15,9 @@ To get a local copy up and running follow these simple example steps.
 ## Setup
 
 1. Clone the Repository or Download the Zip file to set up the website locally. (https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository) This documentation details the steps to take to clone a repository.
-2. Run the live server or simply open index.html with your browser.
+2. [Live Demo Link](https://selmandapanda.github.io/Selma-Hamutenya-Portfolio/)
```

Comment:
- **[OPTIONAL]** This link is not properly documented. A live demo is not a part of the Setup section. The readme template from  Microverse has its own section. This is for legibility. Visitors can see where the live demo quickly. This is the template: [https://github.com/microverseinc/readme-template](https://github.com/microverseinc/readme-template)